adc/ad7779/imxrt/gpio.c

```diff
@@ -38,10 +38,10 @@ static void gpio_setPin(int gpio, int pin, int state)
 	msg.i.size = 0;
 	msg.o.data = NULL;
 	msg.o.size = 0;
+	msg.oid.id = gpio;
```

Comment:
Pedantic - it would be best to initialize msg.oid.port too

Reply:
I've decided against marking all occurrences - it should be easy to find and me spaming 10+ comments about this would only make things worse ;)